sql/flow_config.go

```diff
@@ -38,6 +39,13 @@ var (
 
 var httpClient = &http.Client{}
 
+var semVerRegex = regexp.MustCompile("^(.*)[.](.*)[.](.*)$")
```

Comment:
```suggestion
var semVerRegex = regexp.MustCompile("^(\d{1,})[.](\d{1,})[.](\d{1,})$")
```
nit: more accurate version of the same thing

Reply:
yes, make sense. thanks a lot!